editor/editor_node.cpp

```diff
@@ -734,6 +734,28 @@ void EditorNode::_notification(int p_what) {
 			CanvasItemEditor::ThemePreviewMode theme_preview_mode = (CanvasItemEditor::ThemePreviewMode)(int)EditorSettings::get_singleton()->get_project_metadata("2d_editor", "theme_preview", CanvasItemEditor::THEME_PREVIEW_PROJECT);
 			update_preview_themes(theme_preview_mode);
 
+			if (RenderingDevice::get_singleton() && RenderingDevice::get_singleton()->get_device_type() == RenderingDevice::DEVICE_TYPE_CPU) {
```

Comment:
Why are we doing this check both here and in the initialize function for the OpenGL renderer and the Rendering device? Shouldn't we only be checking this once?

Reply:
Couldn't we just have the message here then so it is always visible?

Reply:
This is EditorNode, but the message should be visible when running an exported project from the command line too.